source/daq/frequency_mask_trigger.cc

```diff
@@ -854,6 +1009,7 @@ namespace psyllid
         a_config.add( "threshold-power-snr-high", new scarab::param_value( a_node->get_threshold_snr_high() ) );
```

Comment:
Aren't we missing the sigma thresholds here?

Reply:
yes, I think we should only export the values corresponding to the threshold type currently set. I've added a switch case based on that assumption (can remove that assumption if you disagree).